README.md

```diff
@@ -1,6 +1,6 @@
-# HID driver for Activision GH Live PS3 and Wii U Guitar devices #
+# HID driver for Activision GH Live PS3, Wii U, and PS4 Guitar devices #
 
-As far as I know this is the first module for the GH Live PS3 and Wii U Guitar devices. I really wanted to play a 6-fret guitar on [Clone Hero](https://clonehero.net/). I could not figure how to get my PS4 dongles to cooperate (yet), but thankfuly it's a different matter for the Wii U ones. Furthermore, the PS3 dongles happen to be exactly the same as the Wii U ones, so this driver supports those as well.
+This driver module supports the GH Live devices for PS3, Wii U, and PS4. This module notably allows you to play songs with the 6-fret guitar in Clone Hero with your PS3, Wii U or PS4 dongle.
```

Comment:
Missig Oxford comma ;-)

Reply:
Oxford comma is used for OR as well as AND?

Reply:
Just read up on Oxford commas. All good. Makes sense!